test/unit/user_test.rb

```diff
@@ -288,6 +288,15 @@ def test_accessible_services
     assert user.authenticated?('new password')
   end
 
+  test 'validate password minimum length' do
```

Comment:
Actually, there is already a very similar test `should weak password must be 6 chars at least` I think the checks could be just added to that test:
```
    assert_not user.valid?
    assert_equal "is too short (minimum is 6 characters)", user.errors.messages[:password].first
```

(but `should` should be removed from the test name I think)

Reply:
correct